sw/airborne/modules/stereocam/stereocam.c

```diff
@@ -76,6 +111,9 @@ extern void stereocam_disparity_to_meters(uint8_t *disparity, float *distancesMe
 
 extern void stereocam_start(void)
 {
+  struct FloatEulers euler = {RadOfDeg(STEREO_BODY_TO_STEREO_PHI), RadOfDeg(STEREO_BODY_TO_STEREO_THETA), RadOfDeg(STEREO_BODY_TO_STEREO_PSI)};
```

Comment:
Just to clarify: If I define it with _unit="deg"_ in the airframe file, I do not have to use the _RadOfDeg_ anymore in stereocam.c anymore?